osu.Game/Screens/Select/Carousel/SetPanelBackground.cs

```diff
@@ -68,5 +70,23 @@ public SetPanelBackground(IWorkingBeatmap working)
                 },
             };
         }
+
+        public partial class PanelBeatmapBackground : Sprite
+        {
+            private readonly IWorkingBeatmap working;
+
+            public PanelBeatmapBackground(IWorkingBeatmap working)
+            {
+                ArgumentNullException.ThrowIfNull(working);
```

Comment:
This aggressive nullcheck is kinda weird, is there some reason for it? I don't really see us aggressively guarding against nulls anywhere else like this.

Reply:
I just copied it from `BeatmapBackgroundSprite` 😅

https://github.com/ppy/osu/blob/a842f79ad49cc98afaef897c85f2f5657ca32029/osu.Game/Beatmaps/Drawables/BeatmapBackgroundSprite.cs#L12-L30

Reply:
Hmm okay. That's weird but also borderline out of scope, I don't care that much.

I think it's fine to keep the implementations separate, but that may be my dislike of bool params talking.